### DISCUSSION TITLE
What's the difference between tldr and cheat pages?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [tldr-pages/tldr#13421] [Q&A] [answered]
Wonder what's the difference between them? I searched for it on the search engine, but it didn't give the result.
![image](https://github.com/user-attachments/assets/158143ef-78c0-4e9a-9cdf-df0a988294d6)

**Comment by spageektti** [comment]
TL;DR pages can be considered a type of cheat pages.

**Comment by kbdharun** [comment, marked as answer]
Hi if you are asking about https://cheat.sh/ it is one of the tldr community clients which allows viewing our pages using CURL (it additionally has support for other sources too).

Our official clients allow for viewing it offline (using caching) and have additional features with highlighting, languages, etc when compared to cheat.sh.

So I would suggest using the CLI clients if you are on a desktop. Or https://tldr.inbrowser.app/ if you are on the Web.
